From 77a54976f2c5e9abc954d5c57688d9f18dff2e25 Mon Sep 17 00:00:00 2001
From: Frederic Boisnard <fredericx.boisnard@intel.com>
Date: Fri, 17 May 2013 17:40:52 +0200
Subject: [PORT FROM MAIN] [WM8958] Allow media process to update AudioComms properties

BZ: 103991

In order to be able to select a specific XML configuration for the
parameter-framework at runtime, the Android property Audiocomms.PFW.ConfPath
must be updated. However the process where the updated can be achieved
is part of the group media, which is not allowed to update this property.

This patch aims to allow media processes to update Android properties
whose name start with "AudioComms" and "audiocomms".

Change-Id: I0d660eb1fb7e87f9c85bce1c7870f7ee36ddf71a
Orig-Change-Id: I96c4e32491a165334a0bd0634f488442a7126ae0
Signed-off-by: Frederic Boisnard <fredericx.boisnard@intel.com>
---
 init/property_service.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/init/property_service.c b/init/property_service.c
index 5fb3ff0..1b5ecd5 100755
--- a/init/property_service.c
+++ b/init/property_service.c
@@ -93,6 +93,8 @@ struct {
     { "persist.security.", AID_SYSTEM,   0 },
     { "persist.service.bdroid.", AID_BLUETOOTH,   0 },
     { "selinux."         , AID_SYSTEM,   0 },
+    { "AudioComms.",       AID_MEDIA,    0 },
+    { "audiocomms.",       AID_MEDIA,    0 },
     { NULL, 0, 0 }
 };
 
-- 
1.7.4.1

